edx-platform/pearson-aws-theme/lms/static/sass/partials/lms/theme/_login-register.scss

```
@@ -1,12 +1,71 @@
+$form-background: #3e5cd2;
+$bg-image: '#{$static-path}/images/login-register.jpg';
+
+
 .login-register-content {
+  min-height: 100vh;
+  background-image: url($bg-image);
+  background-position: left bottom;
+  background-repeat: no-repeat;
+  background-size: cover;
+
   h2 {
-    color: $primary;
+    color: $inverse;
+    font-weight: bold;
   }
 
   .login-register {
+    padding: 20px;
+    margin: auto auto;
+    background-color: $form-background;
+
+    p, .text {
+      color: $inverse;
+    }
+
+    a {
+      color: $inverse;
+
+      &:hover, &:focus {
+        color: $inverse;
+      }
+    }
+
+    // Login and register form fields
+    .form-field {
+      .tip,
```

```suggestion
    .form-field {
       color: $inverse;
       
      .tip,
```
What do you think? It will avoid subsequent definitions of the same rule. @kuipumu

---

I tried that, but setting the same color to all children makes some form field elements  unreadable